homeassistant/components/webostv/notify.py

```diff
@@ -47,7 +47,7 @@ async def async_send_message(self, message: str = "", **kwargs: Any) -> None:
                 await self._client.connect()
 
             data = kwargs.get(ATTR_DATA)
-            icon_path = data.get(CONF_ICON, "") if data else None
+            icon_path = data.get(CONF_ICON, None) if data else None
```

Comment:
```suggestion
            icon_path = data.get(CONF_ICON) if data else None
```
`None` is the default value.
you can also simplify (remove) the data checks by changing it to `data = kwargs.get(ATTR_DATA, {})`